website/docs/language/stacks/deploy/conditions.mdx

```diff
@@ -92,7 +92,7 @@ orchestrate "auto_approve" "safe_plans" {
         reason    = "Plan is destroying ${context.plan.changes.remove} resources."
     }
 
-    # Ensure that the deployment is not your production environment. 
+    # Ensure that the deployment is not to your production environment. 
```

Comment:
Howdy! This is correct as is, this check is ensuring the current deployment is not your production deployment (`context.plan.deployment != deployment.production`):
```suggestion
    # Ensure that the deployment is not your production environment. 
```